app/components/ClientSampleCode.tsx

```diff
@@ -403,5 +403,66 @@ export function ClientSampleCode({
           <Highlight language="sh" code="./a.out" />
         </>
       )
+    case 'c#':
+      return (
+        <>
+          In Visual Studio, create a new C# Console App under File {'>'} New{' '}
+          {'>'} Project. After the project initializes, right click the solution
+          in the Solution Explorer and click Manage NuGet packages for solution.
+          Browse for and install Confluent.Kafka. Copy the following into your
+          Progam.cs file.
+          <Highlight
+            language="cs"
+            code={dedent(String.raw`
+            using Confluent.Kafka;
+
+
+            var config = new ConsumerConfig
+            {
+              SecurityProtocol = SecurityProtocol.SaslSsl,
+              BootstrapServers = "kafka.${domain ?? 'gcn.nasa.gov'}",
+              GroupId = Guid.NewGuid().ToString(),
+              AutoOffsetReset = AutoOffsetReset.Earliest,
+              SaslMechanism = SaslMechanism.OAuthBearer,
+              SaslOauthbearerMethod = SaslOauthbearerMethod.Oidc,
+              SaslOauthbearerTokenEndpointUrl = "https://auth.gcn.nasa.gov/oauth2/token",
+              // Warning: don't share the client secret with others
+              SaslOauthbearerClientId = "${clientId}",
+              SaslOauthbearerClientSecret = "${clientSecret}"
+
+            };
+
+            using (var consumer = new ConsumerBuilder<Ignore, string>(config).Build())
+            {
+              var topics = new List<string> {
+                ${topics.map((topic) => `"${topic}"`).join(`,
+                `)}
+              };
+
+              consumer.Subscribe(topics);
+
+              while (true)
+              {
+                try
+                {
+                  var consumeResult = consumer.Consume();
+                  Console.WriteLine(consumeResult.Message.Value);
+                }
+                catch (Exception ex)
+                {
+                  if (!ex.Message.Contains("Subscribed topic not available"))
+                  {
+                    throw;
+                  }
+                }
+              }
+            }
+            `)}
```

Comment:
Add trailing newline. (It's a POSIX convention that files end with a newline, and since dedent seems to strip off the trailing newline, we need two.)

```suggestion

            `)}
```